### DISCUSSION TITLE
Memory ANE is missing

### DISCUSSION THREAD

**Discussion by jimmymjing** [airsdk/apm#88] [Q&A] [answered]
Hi, this is surely not an issue. I just don't know where to put this piece of info. 
I find that Memory ANE is missing from the repo. For the time being I will just manually add it but it would be nice to have it in the repo.

**Comment by marchbold** [comment, marked as answer]
Yeah, those couple of free ANEs we are still working on getting packages together for. Will have them available in early December.

**Reply by marchbold** [reply]
Hi, The memory ANE is available as an air package now. Let me know if there are any other's missing.
